CHANGELOG.md

```diff
@@ -1,5 +1,123 @@
 # Changelog
 
+## 🌊 1.11.0
+
+- ### Features
+
+  - **New configuration method with `wrangler login` - [jspspike], [pull/1471]**
+
+    `wrangler login` allows you to configure Wrangler to use your user credentials without hunting down an API token in the Cloudflare dashboard. It's straightforward! Just run `wrangler login`, enter your credentials, and you're off to the races.
```

Comment:
Maybe use "authenticate "instead of "configure" and "Cloudflare credentials" instead of "user credentials"